package.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "wl-web-configs",
+    "author": "Workleap",
+    "description": "Common configurations for building web apps and libraries at Workleap",
+    "contributors": [
+        "Alexandre Asselin <alexandre.asselin@gsoft.com>"
+    ],
+    "private": true,
+    "license": "Apache-2.0",
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/Workleap/wl-web-configs.git"
+    },
+    "scripts": {
+        "build": "pnpm -r build",
+        "build:eslint": "pnpm --filter=@workleap/eslint-plugin build",
+        "changeset": "changeset",
+        "clean": "pnpm run /^clean:.*/",
```

Comment:
Instead of delegating the "clean" process to every package, since it's basically always the same command for every packages, you could define the cleaning command directly in the root:

`"clean": "pnpm -r --include-workspace-root --parallel exec pnpm dlx rimraf dist node_modules/.cache",`